app/models/survey/questions_type.rb

```diff
@@ -1,27 +1,27 @@
 class Survey::QuestionsType
-  @@questions_types = {:general => 1}
-  
+  @@questions_types = { multiple_choice: 2, free_response: 9, multi_select: 1 }
```

Comment:
I ended up deciding to use them since it's easier to make the `question_type` setting at the top level rather than trying to take a bottom up approach